src/styles/star.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background: url(../images/star.svg) no-repeat center;
+    display: flex;
```

Comment:
The `display: flex;` property is not necessary here because the `.stars__star` class is not a container for other elements. It's better to remove it to keep your CSS clean and understandable.